src/test/java/com/artipie/rpm/meta/XmlFilelistsTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2019 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.rpm.meta;
+
+import java.nio.file.Path;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+import org.junit.jupiter.api.io.TempDir;
+
+/**
+ * Tests for {@link XmlFilelists}.
+ *
+ * @since 1.0
+ */
+public final class XmlFilelistsTest {
+    // @todo #84:30min This test only creates a temporary file for XmlFilelists,
+    //  now some assertion should be added to verify that this class
+    //  can write `filelists.xml` file correctly. The example of filelists can
+    //  be found at test resources. If #86 is fixed then remove the DisabledOnOs
+    //  annotation.
+    @Test
+    @DisabledOnOs(OS.WINDOWS)
+    public void writesFile(@TempDir final Path temp) throws Exception {
```

Comment:
@victornoel this is a really small PR, why can we add assertions as well?

Reply:
@vatavuk actually it took me more than the allotted time to complete, between understanding existing classes to know what should be done to instantiate all of this and understanding the windows issue... Also, I have no idea how to test for the file content, so I believe it should take the time of one job to write the correct assertions, hence the todo. I'm new to the project, efficiency will increase with time :)